java/client/src/main/java/glide/api/models/commands/geospatial/GeoSearchResultOptions.java

```diff
@@ -52,7 +52,7 @@ public GeoSearchResultOptions(long count) {
         this.isAny = false;
     }
 
-    /** Constructor with count and the 'ANY' keyword. */
+    /** Constructor with count and the {@link #isAny} option. */
```

Comment:
we are linking one arg but not the other. I'm not sure if its necessary to link to the arg or not, but if we do we should probably do it for all args in the doc. Alternatively, I think these comments could be removed as they seem redundant, its already clear by the code itself that it is a constructor with these args

Reply:
Actually, docs on private fields aren't visible to a user, so I'll move them to contstructors.